config/vufind/Primo.ini

```
@@ -165,11 +174,19 @@ labelSections[] = Facets
 ; should prefix the section name with a ~ character to ensure proper loading.
 checkboxSections[] = CheckboxFacets
 
-; Checkbox facets are facets, which are getting displayed as checkboxes
-; pcAvailability is a special facet. It's used to show all records found in
-; PrimoCentral without checking the local availability against a holdingsfile.
+; Checkbox facets are facets, which are getting displayed as checkboxes.
```

I'm not sure where this text came from, but if we want to label this section, maybe we should just use the standard language from facets.ini:

```suggestion
; This section is reserved for special boolean facets.  These are displayed
; as checkboxes.  If the box is checked, the filter on the left side of the
; equal sign is applied.  If the box is not checked, the filter is not applied.
; The value on the right side of the equal sign is the text to display to the
; user.  It will be run through the translation code, so be sure to update the
; language files appropriately.
```

(Unless, of course, this isn't actually true in this context -- but I'm assuming you can use standard checkbox facets, or the special values noted below).